cli/tests/unit/serve_test.ts

```diff
@@ -721,7 +721,7 @@ function createStreamTest(count: number, delay: number, action: string) {
 }
 
 for (const count of [0, 1, 2, 3]) {
-  for (const delay of [0, 1, 1000]) {
+  for (const delay of [0, 1, 25]) {
```

Comment:
We no longer need a 1000ms test here -- there are no more timeouts in Deno.serve that exercise this path, so we can just use 25ms and make tests faster.